tomcat/src/main/java/org/apache/tomcat/util/http/header/HttpCookie.java

```diff
@@ -0,0 +1,27 @@
+package org.apache.tomcat.util.http.header;
+
+import static org.apache.catalina.session.Session.SESSION_ID_IDENTIFICATION;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.apache.catalina.session.Session;
```

Comment:
아! 남기려고 했는데 까먹어서 추가로 하나만 남깁니다.
[구글 자바 컨벤션](https://google.github.io/styleguide/javaguide.html)에 의하면 static import와 non-static import는 각각 한 단락으로 묶여있어야 한답니다~
참고하세요!

> Imports are ordered as follows:
> 
> All static imports in a single block.
> All non-static imports in a single block.
> If there are both static and non-static imports, a single blank line separates the two blocks. There are no other blank lines between import statements.
> 
> Within each block the imported names appear in ASCII sort order. (Note: this is not the same as the import statements being in ASCII sort order, since '.' sorts before ';'.)